docs/reference/faq.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: "FAQ"
+date: 2023-12-19T00:00:00+00:00
+description: "Frequently Asked Questions"
+sidebar_position: 0
+type: "docs"
+---
+
+:::info
+
+Don't see your question covered? Please drop by in [Slack](https://slack.wasmcloud.com), or [file an issue](https://github.com/wasmCloud/wasmCloud.com/issues/new).
+
+:::
+
+### What is wasmCloud?
+
+wasmCloud helps you build and run globally distributed applications on any cloud and any edge, using WebAssembly. For more on our philosophy, see the [intro page](/docs/intro).
+
+### What problem does wasmCloud solve?
+
+Building and running distributed applications is hard.
+
+wasmCloud's design encourages developers to decompose their applications into small, loosely-coupled building blocks of [business logic](/docs/concepts/actors) and [non-functional requirements](/docs/concepts/capabilities). This allows developers to swap out capabilities at runtime without the need to recompile. It allows operators to roll out security patches or upgrades without friction. And it allows applications to dynamically move and scale in response to changing requirements without disrupting a service.
+
+Our goal is to make distributed applications easier, so developers and operators can be more productive.
+
+### What is included in the wasmCloud ecosystem?
+
+wasmCloud provides all the tools to build, deploy, and manage complex distributed applications. In addition to the [host](/docs/concepts/hosts), wasmCloud also has a [CLI](/docs/cli/), a [UI](/docs/cli/ui), and a [deployment manager](/docs/ecosystem/wadm/).
+
+### Is wasmCloud Open Source?
+
+Yes! wasmCloud is a [CNCF Sandbox project](https://www.cncf.io/projects/wasmcloud/). wasmCloud has applied for [Incubating status](https://github.com/cncf/toc/pull/1198).
+
+The source code for wasmCloud is available on [GitHub](https://github.com/wasmCloud), under [Version 2 of the Apache License](https://github.com/wasmCloud/wasmCloud/blob/main/LICENSE).
+
+### Is wasmCloud a WebAssembly runtime?
+
+Not exactly. We sometimes refer to wasmCloud [hosts](/docs/concepts/hosts) as "application runtimes." Hosts do run [WebAssembly components](/docs/concepts/webassembly-components), through the use of [wasmtime](https://wasmtime.dev).
+
+### What is the unit of compute for wasmCloud?
+
+wasmCloud applications are composed of [actors](/docs/concepts/actors) and [capability providers](/docs/concepts/capabilities), which are [linked](/docs/concepts/runtime-linking) together at runtime. Decoupling business logic from capabilities is part of the design philosophy of wasmCloud.
+
+### I see wasmCloud actors are stateless, but I need to store state. Is wasmCloud right for me?
+
+wasmCloud applications can access and store state via [capability providers](/docs/concepts/capabilities). Only actors are stateless.
```

Comment:
Do we want to mention in-memory here? Technically actors can store ephemeral state there right?

Reply:
If you mean like accessing a global variable, I _think_ we discard all in-memory information after each invocation ends so nothing would carry over.

Reply:
Right, actor invocations get fresh state on every invocation. This is by design, so wasmCloud doesn't need to know where to route a particular request; any instance is viable (though NATS leafs will favor local subscribers)